addons/udes_stock/models/stock_move.py

```diff
@@ -132,6 +132,13 @@ def update_orig_ids(self, origin_ids):
                 updated_origin_ids |= previous_mls.mapped("move_id")
             move.move_orig_ids = updated_origin_ids
 
+    def _prepare_move_lines(self, move_lines):
+        """Split the move lines for backordering
+        :param move_lines: Record set of move lines in which to split
+        """
+        for ml in move_lines:
+            ml._split()
```

Comment:
This creates a new move line for the quantity done, don't we want to keep the quantity done in the original move line?

Reply:
Ah no, the comment in the function is misleading, is the other way around.